MetaMask/metamask-mobile
# feat: 1653 first feature flag poc

## **Description**



* This is the actual first implementation of the LaunchDarkly API for feature flags
* This utilizes the `mobileMinimumVersions.appMinimumBuild` feature flag in LaunchDarkly 

## **Related issues**

Fixes: [#1653](https://github.com/MetaMask/mobile-planning/issues/1653)

This PR builds upon work in previous LD setup work: [#10294](https://github.com/MetaMask/metamask-mobile/pull/10294)

## **Manual testing steps**

1. Start the app
2. in `app/components/hooks/MinimumVersions/useMinimumVersions.tsx`
  * Change line: 22 `featureFlags?.mobileMinimumVersions?.appMinimumBuild > currentBuildNumber;`
        to: `featureFlags?.mobileMinimumVersions?.appMinimumBuild > 0;`
3. Restart app & enter password
4. Modal should popup prompting user to update app version

## **Screenshots/Recordings**



### **Before**



### **After**


https://github.com/user-attachments/assets/812fb49d-e79b-4b3f-8f5e-00d3b63a431f



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
